chat-commands.js

```diff
@@ -2479,6 +2479,59 @@ exports.commands = {
 	},
 	blacklisthelp: [`/blacklist [username], [reason] - Blacklists the user from the room you are in for a year. Requires: # & ~`],
 
+	bb: 'battleban',
+	battleban: function (target, room, user) {
+		if (!target) return this.parse('/help battleban');
+
+		let reason = this.splitTarget(target, true);
+		let targetUser = this.targetUser;
+
+		if (!targetUser) {
+			return this.errorReply(`User '${this.targetUsername}' not found.`);
+		}
+		if (!this.can('forcerename', targetUser)) return false;
+		if (targetUser.battleBanned) return this.errorReply(`User '${targetUser.name}' is already banned from battles.`);
```

Comment:
... What does this even mean? Their roombanned from battles? This would be worded better

Reply:
“From battling” is what I meant